src/components/board/SharingTabSidebar.vue

```diff
@@ -13,10 +13,10 @@
 			@input="clickAddAcl"
 			@search-change="asyncFind">
 			<template #noOptions>
-				{{ isSearching ? t('deck', 'Searching for users, groups and circles …') : t('deck', 'No participants found') }}
+				{{ isSearching ? t('deck', 'Searching for users, groups and team …') : t('deck', 'No participants found') }}
```

Comment:
Looks good to me, just wondering if the 'team' here should actually be plural 'teams' to match the others in the sentence? Sounds a bit unnatural to say, but otherwise fine

Reply:
Yes, good catch, thanks :)